cogs/settings.py

```diff
@@ -0,0 +1,26 @@
+import discord
+import functions as func
+
+from discord.ext import commands
+
+class Settings(commands.Cog):
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+        self.emoji = "⚙️"
+        self.invisible = False
+
+    @commands.command()
+    async def togglereminder(self, ctx: commands.Context) -> None:
+        """Turns reminders on for your cooldowns. Make sure you are not blocking DMs."""
```

Comment:
we could have an alias like "qrt" or "qtr".

Reply:
yea maybe qtr